packages/core/src/tools.ts

```diff
@@ -8,10 +8,12 @@ import {
 
 import {
 	debounce as lodashDebounce,
-	merge as lodashMerge,
+	merge as 	lodashMerge,
```

Comment:
did you add this or did prettier? I think you might've accidentally, let's remove it

![image](https://user-images.githubusercontent.com/14989804/99551953-78921f80-298a-11eb-8f47-38d5618ac19a.png)